commec/config/io_parameters.py

```diff
@@ -59,10 +62,15 @@ def __init__(self, args: argparse.ArgumentParser):
         self.db_dir = args.database_dir
 
         # Check whether a .screen output file already exists.
-        if os.path.exists(self.output_screen_file):
-            raise RuntimeError(
-                f"Screen output {self.output_screen_file} already exists. Aborting."
-            )
+        # Note : The only operational difference between force and resume, is that resume is not passed into the search handlers.
+        if os.path.exists(self.output_screen_file) and not (self.config.force or self.config.resume):
+            print(f"Screen output {self.output_screen_file} already exists. \n"
+                  "Either use a different output location, or use --force, --resume to override. "
+                  "\nAborting Screen.")
+            sys.exit(1)
+
+        if self.config.force and self.config.resume:
```

Comment:
Small change we discussed yesterday is switching this over to using the [`mutually_exclusive_group`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group) in `argparse`. Happy to approve after that!

Reply:
Added, ready to go! =D

Reply:
Removed that warning in 505564f8e9786888121b29dee493f857106dd46d since the mutually_exclusive_group should prevent reaching that state